protobuf/psa_purge_key.proto

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
I think similarly than Copy, Purge has meanings that are not yet clear in Parsec's world so I think it is best to leave it out for now 😃

Reply:
Roger, will 🧹 it